editor/editor_asset_installer.cpp

```diff
@@ -553,6 +554,12 @@ void EditorAssetInstaller::_install_asset() {
 			Ref<FileAccess> f = FileAccess::open(target_path, FileAccess::WRITE);
 			if (f.is_valid()) {
 				f->store_buffer(uncomp_data.ptr(), uncomp_data.size());
+				if (target_path.begins_with("res://addons/")) {
+					Vector<String> separated = target_path.substr(13).split("/");
+					if (separated.size() == 2 && separated[1] == "plugin.cfg") {
```

Comment:
Note that this won't work for addons inside subfolders. Check #91337 to see how it can be more reliable.

Reply:
Thanks for this pointer! I'm happy to see an existing use of the magic number 13 :)